apps/sms/test/marionette/lib/views/inbox/accessors.js

```diff
@@ -0,0 +1,37 @@
+var SELECTORS = Object.freeze({
+  main: '#thread-list',
+  firstConversation: '.threadlist-item',
+  smsConversation: '.threadlist-item[data-last-message-type="sms"]',
+  mmsConversation: '.threadlist-item[data-last-message-type="mms"]',
+  navigateToComposerHeaderButton: '#threads-composer-link'
+});
+
+module.exports = {
+  get firstConversation() {
+    return client.helper.waitForElement(
+      SELECTORS.firstConversation
+    );
+  },
+
+  get smsConversation() {
+    return client.helper.waitForElement(
+      SELECTORS.smsConversation
+    );
+  },
+
+  get mmsConversation() {
+    return client.helper.waitForElement(
+      SELECTORS.mmsConversation
+    );
+  },
+
+  waitToAppear: function() {
+    return client.helper.waitForElement(SELECTORS.main);
+  },
+
+  navigateToComposer: function() {
```

Comment:
That's to keep the current test working. Let's change that in a follow up, as we already have a lot of changes in this. I filed [bug 1167103](https://bugzilla.mozilla.org/show_bug.cgi?id=1167103).